CONTRIBUTING.md

```diff
@@ -1,17 +1,39 @@
-# Contributing to the community repo
+# Contributing to the Community Repo
 
-Make a [pull request](https://help.github.com/articles/using-pull-requests) (PR).
+Welcome to the Kubernetes Community contributing guide. We are excited about the prospect of you joining our [community](https://github.com/kubernetes/community]!
 
-Upon successful review, someone will give the PR
-a __LGTM__ (_looks good to me_) in the review thread.
+## Finding Something to Work On
+
+Before you begin, you can look at some places where you can help out:
+
+- [Existing open issues](https://github.com/kubernetes/community/issues)
+- [Open Pull Requests](https://github.com/kubernetes/community/pulls) - Most of the pull requests in this repository are not code, but documentation and process. This is a great place to get started. Even if you do not have the rights to merge you can still help by doing a review.
 
-A [SIG lead](sig-list.md) (or someone with approval powers
+## Steps
+
+1. Make a [pull request](https://help.github.com/articles/using-pull-requests) (PR).
+2. Upon successful review, someone will give the PR
+a __LGTM__ (_looks good to me_) in the review thread.
+3. A [SIG lead](sig-list.md) (or someone with approval powers
 as specified in an OWNERS file) may merge the PR immediately
 with or without an LGTM from someone else.
 Or they may wait a business day to get further feedback from other reviewers.
 
+### Trivial Edits
+
+Each incoming Pull Request needs to be reviewed, checked, have tests run, and then merged. 
```

Comment:
Actually, we don't have tests in this repo. We just have checks on CLA and lgtm.